### ISSUE TITLE
Not showing list of suggestions

### ISSUE THREAD

**Issue by shubham-singh433** [FaridSafi/react-native-google-places-autocomplete#881]
Having a problem with the suggestion list,not showing  tried  giving it the maximum z index
   <GooglePlacesAutocomplete
          placeholder="Search"
          fetchDetails={true}
          onPress={(data, details = null) => {
            // 'details' is provided when fetchDetails = true
            console.log(data, details);
          }}
          query={{
            key: googleAPIKey,
            language: 'en',
          }}
          // currentLocation={true}
          styles={{
            description: {
              fontWeight: 'bold',
            },
            predefinedPlacesDescription: {
              color: '#1faadb',
            },
            listView: {
              color: 'black', //To see where exactly the list is
              zIndex: 1000, //To popover the component outwards
              position: 'absolute',
              top: 45,
            },
            textInputContainer: {
              zIndex: 1000,
              position: 'absolute',
              top: 40,
              width: Dimensions.get('screen').width / 1.5,
            },
          }}
        
        />

**Comment by Beadle85** [comment]
@shubham-singh433 , any luck with this? I'm experiencing the same thing.

**Comment by ScopeSV** [comment]
This worked for me last week. Continued on my app today, and it's not working anymore. Any way to log errors?

**Comment by BenjaminWatts** [comment]
Having same issue, any ideas?

**Comment by vinodsptharsha** [comment]
I had the same issue. It was because of parent container height was not set. You can use flex: 1 on the parent container of GooglePlacesAutocomplete. I hope you guys facing is the same issue.

**Comment by DawitAskabe** [comment]
flex:1 does not work based on how you have layout setup. Ideally we want to be able to set zindex so the result listview shows over every other control in the view.  As example i cant get this to work. The grey you see between the  GooglePlacesAutocomplete inputs is the search result listview. 
Setting zindex does nothing. The 'right' behavior would be the list showing over everything in the view via zindex.



![image](https://github.com/FaridSafi/react-native-google-places-autocomplete/assets/109767729/e9d01547-0c9e-456e-87ba-a0e26d17df3a)

```
   <View style={styles.container}>
        <GooglePlacesAutocomplete
          placeholder="Start"
          fetchDetails={true}
          onPress={console.log('clickeddd')}
          onFail={(error) => console.error(error)}
          query={{
            key: process.env.EXPO_PUBLIC_GOOGLE_API_KEY,
            language: "en",
          }}
          styles={{
            textInputContainer: {
              width: '100%',
              borderWidth: 1,
            },
            textInput: {
              height: 50,
              color: "#5d5d5d",
              fontSize: 16,
              paddingHorizontal: 15,
            },
            predefinedPlacesDescription: {
              color: "#1faadb",
            }, 
            row: {
              backgroundColor: 'grey',
              marginHorizontal: 10,
              zIndex: 11
            },
             listView: {

              zIndex: 1000
            }
          }}
        />

        <GooglePlacesAutocomplete
          placeholder="Destination"
          fetchDetails={true}
          onPress={console.log('clickeddd')}
          onFail={(error) => console.error(error)}
          query={{
            key: process.env.EXPO_PUBLIC_GOOGLE_API_KEY,
            language: 'en',
            components: 'country:us',
          }}
          styles={{
            textInputContainer: {
              borderWidth: 2,
              width: '100%',
              borderWidth: 1,
            },
            textInput: {
              height: 50,
              color: "#5d5d5d",
              fontSize: 16,
              paddingHorizontal: 15,
            },
            row: {
              backgroundColor: 'grey',
              marginHorizontal: 10,
              zIndex: 11
            },
            listView: {
              zIndex: 1000
            }
          }}
        />
      </View>
```

```
const styles = StyleSheet.create({
  container: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    alignContent: 'stretch',
    marginTop: 65,
    height: 130,
    margin: 20
  },
});
```

**Comment by daweimau** [comment]
Thank you @vinodsptharsha for confirming it is a styling issue. This was confusing since absolutely no change was visible.

Reading some other issues, I realised that using this opiton: `suppressDefaultStyles` fixes everything, including the z-axis. This means the problem is somewhere in the default styling.

I confirmed in my case, this is the culprit style:

```ts
            container: {
              flex: 1,
            },
```

This can be overridden:

```ts
        styles={{
          container: {
            flex: undefined,
          },
```
And now everything is fixed (but maybe of course, styling can be improved ;)

**Comment by Seanmclem** [comment]
My problem specifically was that I had opted for the `Places (New) API`, instead of plain old `Places API`. This Library must only support the latter? Either way once I enabled the plain old places API, it worked. 

The real thing that helped from this thread, was adding the `onFail` prop with a `console.error`  on the result. Immediately gave me the proper `api project is not authorized to use this api` hint that I needed. I also adding geocoding.
